cage.c

```diff
@@ -151,6 +152,8 @@ cleanup_primary_client(pid_t pid)
 	} else if (WIFSIGNALED(status)) {
 		wlr_log(WLR_DEBUG, "Child was terminated by a signal (%d)", WTERMSIG(status));
 	}
+
+	return WEXITSTATUS(status);
```

Comment:
I'm not sure this is guaranteed to return something sensible when `WIFEXITED` is false? Maybe we want to guard this value?

Reply:
Edited. Documentation indicates `WEXITSTATUS` is valid only if `WIFEXITED`.
However, Cage now returns exit code 0 when the client is killed by a signal (`SIGKILL` for example). Testing with Bash, exit code is `0x80 | <signal number>` in such a case.

Reply:
Yes. POSIX mandates that shells return a status code >128 when the command is signaled. Bash (and others) returns 128 + WTERMSIG. Maybe we should do the same here?

Reply:
Done. Now Cage returns 137 when the child process is `SIGKILL`ed.